pkg/apis/crd/v1alpha1/types.go

```diff
@@ -291,3 +291,38 @@ type TLSProtocol struct {
 	// SNI (Server Name Indication) indicates the server domain name in the TLS/SSL hello message.
 	SNI string `json:"sni,omitempty"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
```

Comment:
Then the CRD definition shouldn't have status subresource:
```
      subresources:
        status: {}
```

Reply:
OK, I will change it.